Training/2.0 Libraries.md

```diff
@@ -33,16 +38,18 @@ Create a DataFrame using the Pandas library. Your DataFrame should represent the
 [Patient_table.pdf](https://github.com/SoFia2401/gubmes-health-data-analytics/files/7404366/Patient_table.pdf)
 
 
-```
-
+```py
+data = {"A":[50,"Arrythmia"],"B":[60,"Heart Failiure"],"C":[70,"Unstable Angina"]}
+dataFrame = pd.DataFrame(data=data , columns= ["Patient","Age","Heart Condition"])
 ```
 
 ### Task 4
 Normally we would want to be able to extract the information form tables directly into our DataFrame. The Pandas library allows us to do this through one of its functionalities. Find out what this functionality is and read the csv file named 'Training2_example' found in the resources folder or down below:
 
 [Training2_example.csv](https://github.com/SoFia2401/gubmes-health-data-analytics/files/7404392/Training2_example.csv)
 
-```
+```py
+readFile = pd.read_csv(https://github.com/SoFia2401/gubmes-health-data-analytics/files/7404392/Training2_example.csv)
```

Comment:
Correct! Now try Task 4 b) to learn how you can use data frames to perform useful analytics.